File: SoloLearn/Code Coach/Neverland/Neverland.java
import java.util.Scanner;

public class Neverland {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int age = input.nextInt();
        input.nextLine();
        int years = input.nextInt();
        System.out.printf("My twin is %d years old and they are %d years older than me", age + years, years);
        input.close();
    }
} 

File: Code Coach/Neverland/Neverland.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int age = Convert.ToInt32(Console.ReadLine()), years = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("My twin is {0} years old and they are {1} years older than me", age + years, years);
        }
    }
} 